models/migrations/migrations.go

```diff
@@ -345,6 +345,8 @@ var migrations = []Migration{
 	// v195 -> v196
 	NewMigration("Add table commit_status_index", addTableCommitStatusIndex),
 	// v196 -> v197
+	NewMigration("Add Sorting to ProjectIssue table", addProjectIssueSorting),
```

Comment:
The new PR's migration should always be the last one.

Reply:
changed by cd3028a9f0810b6ca563e8582be572f4cc881aeb